revpi_core.c

```diff
@@ -386,26 +451,41 @@ void revpi_core_fini(void)
 	kthread_stop(piCore_g.pUartThread);
 	piIoComm_finish();
 
-	/* reset GPIO direction */
```

Comment:
Please retain this code comment, otherwise it is totally unclear what you're doing in `deinit_sniff_pins()`.